### ISSUE TITLE
Escaped label values are mangled

### ISSUE THREAD

**Issue by twaugh** [DBuildService/dockerfile-parse#7]
```
>>> from dockerfile_parse import DockerfileParser
>>> parser=DockerfileParser('/tmp')
>>> parser.lines=['FROM fedora\n', 'LABEL label=\$VAR\n']
>>> print(parser.labels['label'])
$VAR
>>> assert parser.labels['label'] == '\$VAR'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```

**Comment by twaugh** [comment]
Actually I'm not sure quite how docker handles this, so maybe it isn't an issue after all.

**Comment by twaugh** [comment]
Fixed via #9.
